packages/core/useBreakpoints/index.ts

```diff
@@ -46,27 +48,35 @@ export function useBreakpoints<K extends string>(
 
   const { window = defaultWindow, strategy = 'min-width', ssrWidth = useSSRWidth() } = options
 
-  const ssrSupport = typeof ssrWidth === 'number'
-  const mounted = ssrSupport ? ref(false) : { value: true }
-  if (ssrSupport) {
-    tryOnMounted(() => mounted.value = !!window)
+  const ssrSupport = ref(getCurrentInstance()?.root.isMounted ? typeof ssrWidth === 'number' : false)
+  if (ssrSupport.value) {
+    tryOnMounted(() => ssrSupport.value = !!window)
```

Comment:
Is the intention to disable `ssrSupport` after mount is completed? Shouldn't this be `ssrSupport.value = !window` then?

Reply:
Indeed, good catch